### DISCUSSION TITLE
The cursor line number's color!

### DISCUSSION THREAD

**Discussion by ahmedreda1009** [NvChad/NvChad#2820] [Q&A] [answered]
### Hey everyone,
I'm encountering a slight snag in my Neovim setup that's been puzzling me for a while now that I can't change the cursor line number's color!

i tried to make it in chadrc.lua file like:
```lua
  hl_override = {
    CursorLineNr = { fg = "#000000", bg = "#ff0000" },
  },
```

but it is not working, and then i tried :
```lua
vim.api.nvim_set_hl(0, "CursorLineNr", { fg = "#FFC900", bold = true })

```
but it is not working also

Has anyone else encountered this issue before or have any suggestions on how to resolve it? Any help would be greatly appreciated!

**Comment by siduck** [comment, marked as answer]
https://github.com/NvChad/NvChad/assets/59060246/0b540aca-3fa5-4700-bb87-3d5b1000fc86

works fine here. I hope you are saving file by neovim!

**Reply by ahmedreda1009** [reply]
Disabling all my custom options in options.lua did the trick. While I'm relieved the issue is resolved, I can't pinpoint which specific option was causing the problem. Nonetheless, I'm grateful for your amazing distro—it stands out as the cleanest among the ones I've tried, including lazyvim, astrovim, and lunarvim.

However, there's one hiccup: whenever I open a project like Next.js project with Neovim, it freezes for 5 to 10 seconds before functioning normally.

**Reply by ahmedreda1009** [reply]
I sincerely apologize for any inconvenience caused. Upon further investigation, I've determined that the Tailwind CSS Language Server Protocol (LSP) is the primary culprit behind the 5 to 10 second freezing issue—not anything inherent in the distro. I appreciate your understanding, and please continue your excellent work!

**Reply by siduck** [reply]
@ahmedreda1009 > custom options, i hope you're using v2.5!

**Reply by ahmedreda1009** [reply]
yes I am using 2.5
